flow.js

```diff
@@ -96,6 +97,9 @@ class Flow {
       if(nextFlowStep == 'repair-description' && repairProblemChanged){
         delete state.data['repairProblemBestDescription']
       }
+      if( state.data.communal == 'yes' && state.step =='address') {
```

Comment:
The purpose of this block is to direct the user to the `communal-repairs` page after they have selected an address if they are on the communal repair journey.

A better approach would be to provide the appropriate next step(s) from route.js, see [here](https://github.com/Newarkandsherwood/housing-repairs-online-frontend/pull/85/files#diff-54de87070147bd6483cb396dde9b615c82d0684d7ef8dd228e327ea5b0d36ddcR365) for an example. This approach would remove this specific/special case for communal to be handled differently.